lib/zold/remotes.rb

```diff
@@ -244,13 +247,14 @@ def if_present(host, port)
     end
 
     def load
-      reset unless File.exist?(@file)
-      raw = CSV.read(@file).map do |r|
+      reset unless File.exist?(file)
+      @mtime = File.mtime(file)
```

Comment:
@emilianodellacasa why saving it into an attribute? Why can't we just re-calculate it every time they ask:

```
def mtime
  File.mtime(file)
end
```

Reply:
Actually that was my initial choice, then rubocop marked that as a code smell. Then, thinking about it a little more, I feel that hitting the HD to read the mtime of the file is less efficient that reading it when file is written/read and stored in memory, in case of many requests.

Reply:
@emilianodellacasa but we don't read the file into memory. We read it on every request. We don't cache its content, why caching its mtime?

Reply:
I know we don't read the file, but we are hitting the HD anyway.

However, I removed the attribute and added an _mtime_ method, Rubocop isn't complaining anymore